common/v2/services/Store/BalanceService.tsx

```diff
@@ -123,24 +141,41 @@ export const getAccountsAssetsBalances = async (accounts: StoreAccount[]) => {
 };
 
 export const getAllTokensBalancesOfAccount = async (account: StoreAccount, assets: Asset[]) => {
-  const scanner = getScanner(account.network.nodes[0]);
+  const provider = account.network.nodes[0];
   const assetsInNetwork = assets.filter(x => x.networkId === account.network.id);
   const assetAddresses = getAssetAddresses(assetsInNetwork) as string[];
 
   try {
-    return scanner.getTokensBalance(account.address, assetAddresses);
+    return getTokensBalance(provider, account.address, assetAddresses).then(toBigNumberJS);
```

Comment:
This can possibly be simplified by making the function not async and directly returning the promise instead. The try-catch block wouldn't be necessary (and it seems kind of redundant right now too tbh).

Reply:
I would prefer to keep as much of it the same as possible as this isn't a priority 😄

Reply:
Well, the only difference would be removing the `async` and `try-catch` block. The function would behave exactly the same

Reply:
How come the try-catch isn't needed?

Reply:
```ts
const foo = async (): Promise<any> => {
  throw new Error('bar');
};
```
Is the same as
```ts
const foo = (): Promise<any> => {
  return Promise.reject(new Error('bar'));
};
```